components/dashboard/src/projects/Project.tsx

```diff
@@ -398,26 +398,29 @@ export default function () {
                                                 </a>
                                                 <ItemFieldContextMenu
                                                     className="py-0.5"
-                                                    menuEntries={
+                                                    menuEntries={[
+                                                        {
+                                                            title: "New Workspace ...",
```

Comment:
💭 **thought**: Next steps could include improving the modal to allow branch selection as mentioned in https://github.com/gitpod-io/gitpod/issues/5372, https://github.com/gitpod-io/gitpod/issues/12591, https://github.com/gitpod-io/gitpod/issues/10168, and a [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1673031521923939?thread_ts=1673000615.655599&cid=C01KPEPQYE6) (internal).